src/renderer/components/CustomIcons.tsx

```diff
@@ -143,148 +143,158 @@ export const IconFactoryOld = memo(({ category }: { category: string }) => {
+                        color={accentColor}
+                        transition="0.15s ease-in-out"
+                        viewBox="0 0 4 4"
+                    />
+                );
+            case 'Image (Utilities)':
+                return (
+                    <Icon
+                        alignContent="center"
+                        alignItems="center"
+                        as={libraries.bs.BsGear}
+                        boxSize={boxSize}
+                        color={accentColor}
+                        transition="0.15s ease-in-out"
+                        viewBox="0 0 4 4"
+                    />
+                );
+            case 'Image (Channels)':
+                return (
+                    <Icon
+                        alignContent="center"
+                        alignItems="center"
+                        as={libraries.md.MdAllOut}
+                        boxSize={boxSize}
+                        color={accentColor}
+                        transition="0.15s ease-in-out"
+                        viewBox="0 0 4 4"
+                    />
+                );
+            case 'ONNX':
+                return (
+                    <OnnxIcon
+                        color={accentColor}
+                        transition="0.15s ease-in-out"
+                    />
+                );
+            case 'NCNN':
+                return (
+                    <NcnnIcon
+                        color={accentColor}
+                        transition="0.15s ease-in-out"
+                    />
+                );
+            case 'Utility':
+                return (
+                    <Icon
+                        alignContent="center"
+                        alignItems="center"
+                        as={libraries.bs.BsGearFill}
+                        boxSize={boxSize}
+                        color={accentColor}
+                        transition="0.15s ease-in-out"
+                        viewBox="0 0 4 4"
+                    />
+                );
+            default:
+            // nothing
+        }
 
-    const prefix = icon.slice(0, 2).toLowerCase();
-    const library = (libraries as Partial<Record<string, Partial<Record<string, IconType>>>>)[
-        prefix
-    ];
-    if (!library) {
-        return <OpenCVIcon />;
+        const prefix = icon.slice(0, 2).toLowerCase();
+        const library = (libraries as Partial<Record<string, Partial<Record<string, IconType>>>>)[
+            prefix
+        ];
+        if (!library) {
+            return <OpenCVIcon />;
+        }
+        const libraryIcon = library[icon];
+        return (
+            <Icon
+                alignContent="center"
+                alignItems="center"
+                as={libraryIcon}
+                color={accentColor}
+                height={boxSize}
+                transition="0.15s ease-in-out"
+                viewBox={`0 0 ${boxSize} ${boxSize}`}
+                width={boxSize}
```

Comment:
Why do you use height, width, and viewBox here, while all the icons in the switch statement use boxSize?

Reply:
I have absolutely no idea why I did that originally. I assume it was when I was trying to get the icons to actually scale properly into 4x4 rather than being able to have transparent padding around them (which didn't work) and I just left the code. I can try to standardize it